pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+[id='about-pantheon_{context}']
+= About Pantheon
+
+Pantheon is a content-management system and publishing platform developed by Red Hat. You can use it to publish documents based on AsciiDoc source. After adding your modules to Pantheon, add metadata, preview modules using the look and feel of the Customer Portal, associate the module with a product, publish and unpublish the module.
+
+Pantheon provides:
+
+    A GUI for managing modules and assemblies, including the ability for authors to search for specific content and other workflow features that are more user-friendly than those currently available
+    Metadata editing to optimize content for search engines
+    A module library to enable content reuse
+    A visual preview of a module so you can see how it will appear on the customer portal
+
+Pantheon does not:
+
+    Determine the appearance of content
+    Replace git as a source control system
```

Comment:
I'd either use Git, the "proper" name of the tool, or mark it up in monospace to indicate a literal command (`git`).

Reply:
Fixed